install/kubernetes/helm/istio/charts/prometheus/values.yaml

```diff
@@ -58,3 +58,7 @@ service:
 
 security:
   enabled: true
+
+configmap:
```

Comment:
I don't think this is a good idea to put more parameters in value.yaml, why do you need to limit the namespaces for Prometheus scrapes from?

Reply:
We already have a prometheus instance in the cluster which is scraping the annotated applications.
We're happy with a separate Prometheus instance that is only taking care of istio-system metrics, but we don't want it to scrape all the annotated apps.
And it's tricky because we're using helm to manage Istio, so I can't find a way to manipulate that configmap with helm, because the "owner" of it is the istio chart itself.
Modifying it manually is not an option at the moment, neither configure a federated instance of Prometheus.

Reply:
Can you append the [Istio specific scrape configs](https://github.com/istio/istio/blob/2a99d6ac984718785772919c72513f5a85251d37/install/kubernetes/helm/istio/charts/prometheus/templates/configmap.yaml#L17-L281) to your existing Prometheus instance's configuration?

Reply:
thanks for your suggestion @ayj .
We piped this option up to be examined in the future setup of our system.
Unfortunately not an option in the near future. 
But yes, definitely something that we want to try, thank you.